apps/navstar_overlay/src/navstar_overlay_lashup_kv_listener.erl

```diff
@@ -106,52 +107,61 @@ terminate(_Reason, _State, _Data) ->
+%%--------------------------------------------------------------------------------
 %% State transition
-%% 
+%%-------------------------------------------------------------------------------- 
 %%                                                                lashup_event
 %%                                                                  +------+
 %%                                                                  |      |
-%%  +--------------+              +----------------+              +-+--------+                +-------------+
-%%  |              |              |                |              |          |                |             |
-%%  |              | lashup_event |                | lashup event |          |   timeout      |             |
-%%  | unconfigured +------------> |    applying    +------------> | batching +--------------> | configuring |
-%%  |              |              |  first_config  |              |          |                |             |
-%%  |              |              |                |              |          | <--------------+             |
-%%  |              |              |                |              |          |  lashup_event  |             |
-%%  +--------------+              +----------------+              +----------+                +-------------+
+%%  +--------------+              +----------------+              +-+--------+
+%%  |              |              |                |              |          |
+%%  |              | lashup_event |                | lashup event |          |
+%%  | unconfigured +------------> |  configuring   +------------> | batching |
+%%  |              |              |                |              |          |
+%%  |              |              |                | <------------|          |
+%%  |              |              |                |   timeout    |          |
+%%  +--------------+              +----------------+              +----------+
 %%
-%%--------------------------------------------------------------------------------------------------------------
+%%--------------------------------------------------------------------------------
 
 unconfigured(info, EventContent, Data) ->
-    {next_state, applying_first_config, Data, {next_event, internal, EventContent}}.
+    {next_state, configuring, Data, {next_event, internal, EventContent}}.
 
-applying_first_config(internal, 
-            _EventContent = {lashup_kv_events, Event = #{key := [navstar, overlay, Subnet], ref := Ref}},
-            _Data = #data{ref = Ref}) when is_tuple(Subnet), tuple_size(Subnet) == 2 ->
-    lager:info("Applying first config: ~p~n", [Event]),
-    navstar_overlay_configure:start_link(reply, Event),
+configuring(internal, {lashup_kv_events, NewEvent = #{key := [navstar, overlay, Subnet], ref := Ref}},
+            #data{ref = Ref, events = OldEvent}) when is_tuple(Subnet), tuple_size(Subnet) == 2 ->
+    DeltaEvent = get_delta_event(NewEvent, OldEvent),
```

Comment:
We may not be able to calculate delta using ingest_update because of batching. navstar_overlay reference of old value might be different from lashup_kv old value.